pkg/scheduler/algorithm/well_known_labels.go

```diff
@@ -17,47 +17,47 @@ limitations under the License.
 package algorithm
 
 const (
-	// When feature-gate for TaintBasedEvictions=true flag is enabled,
-	// TaintNodeNotReady would be automatically added by node controller
-	// when node is not ready, and removed when node becomes ready.
+	// TaintNodeNotReady would be automatically added by node controller,
+	// when feature-gate for TaintBasedEvictions=true flag is enabled
+	// and node is not ready, and removed when node becomes ready.
 	TaintNodeNotReady = "node.kubernetes.io/not-ready"
 
 	// DeprecatedTaintNodeNotReady is the deprecated version of TaintNodeNotReady.
 	// It is deprecated since 1.9
 	DeprecatedTaintNodeNotReady = "node.alpha.kubernetes.io/notReady"
 
-	// When feature-gate for TaintBasedEvictions=true flag is enabled,
 	// TaintNodeUnreachable would be automatically added by node controller
-	// when node becomes unreachable (corresponding to NodeReady status ConditionUnknown)
+	// when feature-gate for TaintBasedEvictions=true flag is enabled and
```

Comment:
I think for sake of consistency we can start with `TaintNodeUnreachable will be added when node becomes..`